### ISSUE TITLE
States not localized in user List

### ISSUE THREAD

**Issue by ajeitler** [bigbluebutton/bigbluebutton#9005]
**Describe the bug**
On a german localized installation of BBB a screan reader reads u user's state in the user list still in English. 

**To Reproduce**
1. Take a screen reader like nvda windows or voiceover on apple devices and navigate to the user list.
2. The screen Reader reads the user's name etc. plus the state in Englisch.

**Expected behavior**
If I turn the language to - for example - german, the entire UI (even the parts only read for screen readers) should be localized to german.

**Actual behavior**
Alternative Text for state icons should be read in selected language.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser Chrome/Safri/Firefox
 - Version [latest]

**Additional context**
May be a possible solution: I realized, that there are two different Version of language ressource files in bigbutton-client/locale, "de" and "de_DE". The file in "de" still contains English strings. Maybe just copying the de_DE variant to de would solve the issue?
I have no running dev enviroment to check this, sorry.
If this fixes the issue, there is till the question why the rest of the german strings were using de_DE, while other de.
